pkg/operator2/operator.go

```diff
@@ -41,13 +41,15 @@ import (
 var deploymentVersionHashKey = operatorv1.GroupName + "/rvs-hash"
 
 const (
-	clusterOperatorName = "authentication"
-	targetName          = "openshift-authentication"
-	targetNameOperator  = "openshift-authentication-operator"
-	globalConfigName    = "cluster"
+	clusterOperatorName     = "authentication"
+	targetName              = "oauth-server"
```

Comment:
Why `oauth-server` vs `integrated-oauth-server` to match existing APIs like authentication `spec.type`?